comp_sci_gender_bias/getters/dfe_school_data.py

```diff
@@ -0,0 +1,98 @@
+from comp_sci_gender_bias import PROJECT_DIR
+import pandas as pd
+import pathlib
+from comp_sci_gender_bias.utils.process_pandas import cols_replace_space_and_lowercase
+
+
+DFE_DIR = PROJECT_DIR / "inputs/data/dfe_school_info/"
+DFE_2021_DIR = DFE_DIR / "2020-2021/"
+DFE_2019_DIR = DFE_DIR / "2018-2019/"
+SCHOOL_INFO_RENAME_COLS = {
+    "indicates_whether_it's_a_mixed_or_single_sex_school": "school_sex_type"
+}
+CENSUS_RENAME_COLS = {
+    "percentage_of_pupils_eligible_for_fsm_at_any_time_during_the_past_6_years": "percentage_pupils_fsm_past_6_years",
+}
+KS4_RENAME_COLS = {
+    "average_attainment_8_score_per_girl_-_gcse_only": "average_girls_attainment_8_gcse_score",
+    "average_attainment_8_score_per_boy_-_gcse_only": "average_boys_attainment_8_gcse_score",
+    "%_of_boys_achieving_strong_9-5_passes_in_both_english_and_mathematics_gcses_": "percent_boys_strong_9to5_passes_eng_math_gcses",
+    "%_of_girls_achieving_strong_9-5_passes_in_both_english_and_mathematics_gcses_": "percent_girls_strong_9to5_passes_eng_math_gcses",
+}
```

Comment:
small inconsistency: "percent" and "percentage"

Reply:
I've changed them all to "percentage"